sunpy/coordinates/frames.py

```diff
@@ -257,25 +275,30 @@ class Heliocentric(SunPyBaseCoordinateFrame):
     This frame may either be specified in Cartesian or cylindrical
     representation. Cylindrical representation replaces (x, y) with (rho, psi)
     where rho is the impact parameter and psi is the position angle in degrees.
```

Comment:
```suggestion
    where ``rho`` is the impact parameter and ``psi`` is the position angle.
    ``psi`` is measured relative to the west limb, rather than solar north, so is shifted
    by 90 degrees compared to the convention of the Heliocentric Radial system.
```